doc/source/tutorial/stats/continuous_genhyperbolic.rst

```diff
@@ -0,0 +1,136 @@
+
+.. _continuous-genhyperbolic:
+
+Generalized Hyperbolic Distribution
+======================================
+
+The Generalized Hyperbolic Distribution is defined as the normal variance-mean mixture with Generalized Inverse Gaussian distribution as mixing distribution.
+The "hyperbolic" characterization refers to the fact that shape the log-probability distribution can be described as an hyperbola. Hyperbolic distributions are sometime referred as semi-fat tail because their probability density decrease slower than "sub-hyperbolic" distributions (e.g. normal distribution, whose log-probability decreases quadratically), but faster than other "extreme value" distributions (e.g. `pareto` distribution, whose log-probability decreases logarithmically).
+
+
+Functions
+---------
+
+Different parameterization exists in literature, scipy implementing the location-scale as for Prause (1999)
+
+.. math::
+
+   \begin{eqnarray*} f(x, p, \hat{\alpha}, \hat{\beta}, \text{location}, \text{scale}) &=&
+            \frac{(\hat{\alpha}^2 - \hat{\beta}^2)^{\lambda/2}}
+            {\sqrt{2\pi}\hat{\alpha}^{\lambda-0.5}
+            K_\lambda\Big(\sqrt{\hat{\alpha}^2 - \hat{\beta}^2}\Big)}
+            e^{\hat{\beta}(x)} \times \frac{K_{\lambda - 1/2}
+            (\hat{\alpha} \sqrt{1 + x^2})}
+            {(\sqrt{1 + x^2})^{1/2 - \lambda}} \end{eqnarray*}
+
+for:
+
+- :math:`x, \lambda \in \mathbb{R}`
+- :math:`|\hat{\beta}| \lt \hat{\alpha}` if :math:`p \ge 0`
+- :math:`|\hat{\beta}| \lt \hat{\alpha}` if :math:`p = 0`
+- :math:`|\hat{\beta}| \leq \hat{\alpha}` if :math:`p \lt 0`
+- :math:`K_{\lambda}(.)` denotes the modified Bessel function of the second kind and order :math:`p` (`scipy.special.kn`)
+
+which derives from the original parameterization in  Barndorff (1978) by setting:
+
+- :math:`p = \lambda`
+- :math:`\hat{\alpha} = \delta \alpha`
+- :math:`\hat{\beta} = \delta \beta`
+- :math:`\text{location} = \mu`
+- :math:`\text{scale} = \delta`
+
+
+Random variates for the `scipy.stats.genhyperbolic` can be efficiently sampled from the above-mentioned normal variance-mean mixture where `scipy.stats.geninversegauss` is parametrized as :math:`GIG = f(p = p, b = \Big(\hat{\alpha}^2 - \hat{\beta}^2\Big)^{1/2}, \text{loc} = \text{location}, \text{scale} = \Big(\hat{\alpha}^2 - \hat{\beta}^2\Big)^{ - 1/2})` so that :math:`GH(p, \hat{\alpha}, \hat{\beta}) = \hat{\beta} GIG + \sqrt{GIG} N(0,1)`
```

Comment:
```suggestion
Random variates for the `scipy.stats.genhyperbolic` can be efficiently sampled from the above-mentioned normal variance-mean mixture where `scipy.stats.geninvgauss` is parametrized as :math:`GIG = f(p = p, b = \Big(\hat{\alpha}^2 - \hat{\beta}^2\Big)^{1/2}, \text{loc} = \text{location}, \text{scale} = \Big(\hat{\alpha}^2 - \hat{\beta}^2\Big)^{ - 1/2})` so that :math:`GH(p, \hat{\alpha}, \hat{\beta}) = \hat{\beta} GIG + \sqrt{GIG} N(0,1)`
```